src/Daemon.cpp

```diff
@@ -111,6 +111,25 @@ Response execute_command(const IndexCommand &cmd, Task *task,
     return Response::ok();
 }
 
+Response execute_command(const ConfigGetCommand &cmd, Task *task,
+                         const DatabaseSnapshot *snap) {
+    if (cmd.keys().empty()) {
+        return Response::config(snap->get_config().get_all());
+    }
+    std::map<std::string, uint64_t> vals;
```

Comment:
```suggestion
    std::unordered_map<std::string, uint64_t> vals;
```

This change requires `#include <unordered_map>`!